### PR TITLE
Add combine spouse registrations option

### PR DESCRIPTION
**Pull request by canac**
Add an option to automatically combine spouse's registrations.

### CODE REVIEW

**Comment by hlbraddock** [comment]
@canac Has this been tested?

**Comment by canac** [comment]
No, it has not.

**Comment by hlbraddock** [comment]
Is it ready to be tested?

**Comment by canac** [comment]
Yes, after rebasing off of master.

**Comment by hlbraddock** [comment]
I've tested this. @canac +1

**Comment by hlbraddock** [comment]
@OzzieOrca Can you review today? We have a hard target deploy date/time of next Monday around 12PM.

File: app/views/eventDetails/regOptions.html [hunk 1]
```diff
@@ -34,6 +34,17 @@ <h2 class="page-title border" translate>Registration Options</h2>
          popover-append-to-body="true">
       </i>
     </div>
+
+    <div class="checkbox">
+      <label><input type="checkbox" ng-model="conference.combineSpouseRegistrations">
+        <translate>Combine spouse registrations</translate>
+      </label>
+      <i class="question-popover fa fa-question-circle"
```
Comment (dsgoers) [inline app/views/eventDetails/regOptions.html]: @canac I think we should add something here explicitly communicating this field doesn't do anything unless the spouse is on a family registration.

Comment (dsgoers) [inline app/views/eventDetails/regOptions.html]: @hlbraddock and I discussed this

Comment (canac) [inline app/views/eventDetails/regOptions.html]: ✔️  I changed the info message to communicate that.

**Pull request merged**